specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-03-01/examples/CreateExtension.json

```diff
@@ -99,6 +100,7 @@
             "omsagent.env.clusterName": "clusterName1"
           },
           "provisioningState": "Creating",
+          "installedVersion": "0.1.4",
```

Comment:
installedVersion value will continue to be null, until the provisioningState reaches a terminal state.  Since it is 'Creating' now, installedVersion will be null.